projects/utils-ogc/src/lib/wps/wps200/wps_marshaller_2.0.0.ts

```diff
@@ -113,7 +113,8 @@ export class WpsMarshaller200 implements WpsMarshaller {
   }
 
   executeUrl(baseurl: string, processId: string): string {
-    return `${baseurl}?service=WPS&request=Execute&version=2.0.0&identifier=${processId}`;
+    // return `${baseurl}?service=WPS&request=Execute&version=2.0.0&identifier=${processId}`;
+    return baseurl;
```

Comment:
What should this function do?
You don't need this function in the current implementation.

Reply:
We have two classes that implement the `WpsMarshaller` interface ([here](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/master/projects/utils-ogc/src/lib/wps/wps_datatypes.ts#L100)). The interface enforces a method `executeUrl`. 

 - For WPS 1.0.0, this does return a url with a bunch of url-parameters. 
 - For WPS 2.0.0, no url-parameters are required.

The user should not have to know about this difference - I think that for the user the same interface should be visible no matter the implementation details behind the interface. 
Of course we could remove `executeUrl` from the 2.0-marshaller, but that would make the API inconsistent for the user ... and defeat the purpose of why we created a unified API for both protocols in the first place.

Reply:
So in case of WPS 2.0.0 it is enough to post on the `baseurl` and for WPS 1.0.0 you must provide the full URL `${baseurl}?service=WPS&request=Execute&version=2.0.0&identifier=${processId}`?

https://github.com/dlr-eoc/ukis-frontend-libraries/blob/49c552de5f1c4659fe39b80282108989f80698a1/projects/utils-ogc/src/lib/wps/wpsclient.ts#L245

It that's what the WPS Spec wants then ok :D